specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2024-01-01-preview/Recommendations.json

```diff
@@ -297,66 +295,39 @@
       },
       "type": "object"
     },
-    "Content": {
-      "description": "Content section of the recommendation.",
+    "RecommendedSuggestion": {
+      "description": "What suggestions should be taken to complete the recommendation.",
       "properties": {
+        "suggestionTypeId": {
+          "description": "Id of the suggestion type.",
+          "type": "string"
+        },
         "title": {
-          "description": "Title of the content.",
+          "description": "Title of the suggestion.",
           "type": "string"
         },
         "description": {
-          "description": "Description of the content.",
-          "type": "string"
-        }
-      },
-      "required": [
-        "title",
-        "description"
-      ],
-      "type": "object"
-    },
-    "Instructions": {
-      "description": "Instructions section of a recommendation.",
-      "properties": {
-        "actionsToBePerformed": {
-          "description": "What actions should be taken to complete the recommendation.",
+          "description": "Description of the suggestion.",
           "type": "string"
         },
-        "recommendationImportance": {
-          "description": "Explains why the recommendation is important.",
+        "action": {
+          "description": "Action of the suggestion.",
           "type": "string"
         },
-        "howToPerformActionDetails": {
-          "description": "How should the user complete the recommendation.",
-          "type": "string",
+        "additionalProperties": {
```

Comment:
ARM requires Resource provider teams to define the types explicitly. This is to ensure good customer experience in terms of the discoverability of concrete type definitions. 

refs: https://github.com/Azure/azure-openapi-validator/blob/main/docs/avoid-additional-properties.md#avoidadditionalproperties